.travis/build-deps.sh

```diff
@@ -27,7 +27,7 @@ build_mbedtls () {
 
 download_openssl () {
     if [ ! -f "download-cache/openssl-${OPENSSL_VERSION}.tar.gz" ]; then
-        wget -P download-cache/ \
+        wget -P download-cache/ --no-check-certificate \
```

Comment:
for building openvpn master branch we need "xenial" mingw. after I added "xenial" repo, somehow it broke https downloads.

I haven't yet figured out why (and probably, we should just use --no-check-certificate until travis will introduce native support for xenial images, or ... migrate to docker)